pages/integrations/authorization.mdx

```diff
@@ -215,9 +215,9 @@ Including the `Content-Type: application/x-www-form-urlencoded` header, the requ
 | grant_type    | yes      | string | This must be set to _**refresh_token.**_                                                                 |
 | refresh_token | yes      | string | A valid refresh token.                                                                                   |
 
-### Step 8
+The API POST to the token endpoint for retrieving a new access token and refresh token pair will return a JSON response that includes the fields `"access_token"` and `"refresh_token"`. These tokens can be used in the same way as the ones generated in Step 5.
```

Comment:
Consolidated Steps 7 and 8 since they both explained about generating and using the `"refresh_token"`